### ISSUE TITLE
rust-analyzer is not showing variable names in LSP when used with neovim or vs code.

### ISSUE THREAD

**Issue by whereistejas** [rust-analyzer/rust-analyzer#10554]
Previously, code completion used to show all the variables present in the current method/code block, however since the latest release it is no longer showing them.

**Comment by lnicola** [comment]
I can't reproduce on 7d1015b8d177a02dff03d2c70bc74e5d4ed34335. Can you try in a new project?

**Comment by whereistejas** [comment]
It's working in a new project? Interesting! Do I need to clean/clear some kind of cache?

**Comment by whereistejas** [comment]
Nope, the server is not crashing. I'm able to see all the other stuff like method_names and macros in my code_completion dropdown. Only variables are missing :disappointed: .

**Comment by felipesere** [comment]
I have the same issue on a brand new Neovim 0.6 and `2021-11-29 stable`...
Not sure how I can get to the log from nvim?

**Comment by tjkirch** [comment]
Same here with Neovim 0.6.  Out of curiosity, I tried with rust-analyzer nightly and a bunch of rust-analyzer releases going back to the start, and they gave the same error.  (I have no experience in this area, I was just trying to narrow down the issue.)

**Comment by sondr3** [comment]
I'm suddenly getting the same error with neovim 0.6 and the latest rust-analyzer as well, not sure where the error lies but it seemingly happened out of nowhere using the built-in LSP and `rust-tools`, I tried compiling neovim from git but it didn't work either.

**Comment by ChayimFriedman2** [comment]
Probably irrelevant by now.
